device_doctor/tool/build.bat

```diff
@@ -1,8 +1,8 @@
-@ECHO off
-REM Copyright 2020 The Flutter Authors. All rights reserved.
-REM Use of this source code is governed by a BSD-style license that can be
-REM found in the LICENSE file.
+:: Copyright 2020 The Flutter Authors. All rights reserved.
+:: Use of this source code is governed by a BSD-style license that can be
+:: found in the LICENSE file.
 
+@ECHO off
```

Comment:
Why echo off? I find there to be an advantage that magic scripts like this print out their process. This makes it easier for developers to understand why a failure may have occurred based on CI output.

Reply:
Turn echo on.